homeassistant/components/webostv/__init__.py

```diff
@@ -93,6 +97,28 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+def read_client_keys(config_file):
+    """Read legacy client keys from file."""
+    if not os.path.isfile(config_file):
+        return
+
+    # Try to parse the file as being JSON
+    with open(config_file, encoding="utf8") as json_file:
+        try:
+            client_keys = json.load(json_file)
```

Comment:
We have this util:
https://github.com/home-assistant/core/blob/59cea56e170d9e55e3c77b0be5dd47ded75919ce/homeassistant/util/json.py#L26

Reply:
The ` load_json` from util will log an exception and here we need to silently abort and try parsing the file as sqlite. We can also remove this part since migration to sqlite was introduced in core 2021.4.0b0

Reply:
I was thinking we catch HomeAssistantError and continue.

Reply:
Sorry, I didn't noticed we logged besides raising. That's not good for a tool to both log and raise.

👍